From 5d8e4126b6f8e87ac9878531485f3fb7199023bf Mon Sep 17 00:00:00 2001
From: Ivan Zaitsev <ivanzaitsev@meta.com>
Date: Fri, 8 Mar 2024 00:50:45 +0000
Subject: [PATCH] Fixup test_trace_rules (#121351)

Summary:
Fixes
https://www.internalfb.com/intern/testinfra/diagnostics/7599824578133672.281475099376195.1709732674/

(for some reason this test didn't run in OSS)?

Reached out to Yanbo Liang for additional context:
 {F1465435684}

Test Plan:
Local:
https://www.internalfb.com/intern/testinfra/testconsole/testrun/16325548673376150/

Differential Revision: D54605075

Pull Request resolved: https://github.com/pytorch/pytorch/pull/121351
Approved by: https://github.com/malfet, https://github.com/yanboliang
---
 torch/_dynamo/trace_rules.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/torch/_dynamo/trace_rules.py b/torch/_dynamo/trace_rules.py
index 3f583ab75dded..69c5648cc14f2 100644
--- a/torch/_dynamo/trace_rules.py
+++ b/torch/_dynamo/trace_rules.py
@@ -793,6 +793,7 @@
         "torch._C._last_executed_optimized_graph",
         "torch._C._len_torch_dispatch_stack",
         "torch._C._len_torch_function_stack",
+        "torch._C._linalg._linalg_eigvals",
         "torch._C._linalg.linalg_cholesky_ex",
         "torch._C._linalg.linalg_cholesky",
         "torch._C._linalg.linalg_cond",
